runtime/doc/Makefile

```diff
@@ -6,32 +6,14 @@
 
 AWK = awk
 
-# Set to $(VIMTARGET) when executed from src/Makefile.
-VIMEXE = vim
-
 DOCS = $(wildcard *.txt)
 HTMLS = $(DOCS:.txt=.html)
 
 .SUFFIXES:
 .SUFFIXES: .c .o .txt .html
 
-all: tags html
-
-# Use Vim to generate the tags file.  Can only be used when Vim has been
-# compiled and installed.  Supports multiple languages.
-vimtags: $(DOCS)
-	$(VIMEXE) -u NONE -es -c "helptags ++t ." -c quit
-
-# Use "doctags" to generate the tags file.  Only works for English!
-tags: doctags $(DOCS)
-	./doctags $(DOCS) | LANG=C LC_ALL=C sort >tags
-	uniq -d -2 tags
-
-doctags: doctags.c
```

Comment:
This is an old alternative to using vim itself to generate the helptags. Unnecessary for us.